pom.xml

```diff
@@ -43,7 +43,9 @@
     </licenses>
 
     <properties>
-        <version.org.kie>7.1.0-SNAPSHOT</version.org.kie>
+        <version.org.kie>7.36.0-SNAPSHOT</version.org.kie>
+        <version.org.jboss.errai>4.7.0-SNAPSHOT</version.org.jboss.errai>
```

Comment:
do we want really to have here SNAPSHOT of Errai?

Reply:
@mareknovotny we override this property from `SetErraiVersion.groovy` script

Reply:
ok, and there is 4.6.0.Final or something else? I mean if we don't use here SNAPSHOT we can directly set here that 4.6.0.Final, but I am just asking what and why ;)

Reply:
@mareknovotny you are right, there is a `4.6.0.Final` so pointing it there https://github.com/errai/errai/blob/b8c4beeef9bf6c8d93e0f1d14003bcdad02bcf34/pom.xml#L14